add Latvian language

darksworm opened the pull request:
<!--

Have you read Code of Conduct? By filing an Pull Request, you are expected to comply with it, including treating everyone with respect: https://github.com/hovancik/stretchly/blob/master/CODE_OF_CONDUCT.md

-->

Issue: #868
<!-- Link to relevant issue. All PRs should be associated with an issue -->

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

<!-- If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so. -->

- [x]  issue was opened to discuss proposed changes before starting implementation. It is important do discuss changes before implementing them (Why should we add it? How should it work? How should it look? Where will it be? ...).
- [x]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [x]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [x]  app version number was not changed.
- [x]  all new code has tests to ensure against regressions.
- [x] `npm run lint` reports no offenses.
- [ ] `npm run test` is error-free.  - **it actually fails but not due to these changes.**
- [x]  README and CHANGELOG were updated accordingly.
- [x]  after PR is approved, all commits in it are [squashed](https://gitbetter.substack.com/p/how-to-squash-git-commits)

### Description of the Change

<!--

We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts.

-->

Added latvian translations JSON file from weblate and added Latvian to the language selector in settings.


### Verification Process

After making the changes I walked through the application (yes, also the contributor settings) and re-read all the translations to make sure they are OK. 

Some are not perfect, due to how word ending formation differs in Latvian vs English (they are very context dependent in Latvian and differ a lot).

For example in english "for 10 minutes", "about 10 minutes" uses the same form of the word "minute". In Latvian, those become "uz 10 minūtēm", "aptuveni 10 minūtes" - having these forms correct in Stretchly is currently not possible.

<!--

What process did you follow to verify that your change has the desired effects?

- How did you verify that all new functionality works as expected?
- How did you verify that all changed functionality works as expected?
- How did you verify that the change has not introduced any regressions?

Describe the actions you performed (e.g., buttons you clicked, text you typed, commands you ran, etc.), and describe the results you observed.

-->

hovancik commented:
Hi @darksworm , I've merged your changes from Weblate here: #867. They have integration, so no need to include JSON here. You should also add the language option to Welcome.html. I use flags from here: https://github.com/HatScripts/circle-flags

If you feel that some other form would be better in Latvian, feel free to change it as you see fit. I do the same when translate to Slovak.

darksworm committed:
add Latvian language to preferences and welcome screen

darksworm committed:
Merge branch 'trunk' into trunk

darksworm commented:
> Hi @darksworm , I've merged your changes from Weblate here: #867. They have integration, so no need to include JSON here. You should also add the language option to Welcome.html. I use flags from here: https://github.com/HatScripts/circle-flags
> 
> If you feel that some other form would be better in Latvian, feel free to change it as you see fit. I do the same when translate to Slovak.

Really cool that weblate creates pull requests automatically! 

I've removed the JSON file and added the Latvian language option to `welcome.html`.
![image](https://user-images.githubusercontent.com/9987548/108741158-16275d00-753f-11eb-8c05-a2bff35b266b.png)
